docs/code/BSTLD.py

```diff
@@ -14,8 +14,7 @@
 """Authorize a Client Instance"""
 from tensorbay import GAS
```

Comment:
`docs/source/quick_start/examples/get_label_statistics.rst` need to modify

Reply:
`docs/source/features/version_control/getting_started_with_version_control.rst` need to modify

Reply:
`docs/source/reference/label_format/CommonLabelProperties.rst` and other files under label_format